BankChain/app/src/main/res/values/styles.xml

```diff
@@ -14,6 +14,8 @@
         <item name="windowNoTitle">true</item>
     </style>
 
+
+
```

Comment:
Please try to check what files and chunks you stage before you commit, it keeps the history clean from this.